src/cf-forms.less

```diff
@@ -56,15 +60,32 @@
         <label class="form-label-header">
             Form label header
         </label>
+    - name: Form label helper text
+      markup: |
+        <label class="form-label-helper-text">
+            Form label helper text
+        </label>
   tags:
     - cf-forms
 */
 
 .form-label-header {
-    .h5();
+    .h4();
```

Comment:
@sonnakim can you clarify which element the new standard affects in this case? Just the default label style or the other case Scott mentioned, too?